subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_internal.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define SW_SWITCH_RX 0
+#define SW_SWITCH_TX 1
+#define SW_SWITCH_PHY_1M 0
+#define SW_SWITCH_FLAGS_DONTCARE 0
```

Comment:
Looking more into how these are used, it looks like the RX/TX values are used for the `dir_curr` and `dir_next` exclusively. 
Similarly, the `PHY_1M` is used exclusively for the `phy_next` and the `FLAGS_DONTCARE`. 

Based on the naming, this isn't clear. I suggest to, either in this or future PR, either
1) Move these into enums and use the enum types in  `sw_switch` 
2) Add an infix in the macro to make their use more clear, e.g. `SW_SWITCH_DIR_RX` and `SW_SWITCH_DIR_TX`

Reply:
I'd like to avid further factorization of this code in the PR. The goal of the PR is a bit different. I suggest to create a github issue for code improvements and store there all of those comments.